python/lsst/faro/measurement/TractTableMeasurementTasks.py

```diff
@@ -21,14 +21,66 @@
 
 from lsst.pex.config import Config, Field
 from lsst.pipe.base import Struct, Task
+from lsst.pipe.tasks.configurableActions import ConfigurableActionStructField
 from lsst.verify import Measurement, Datum
 
+import lsst.faro.utils.selectors as selectors
 from lsst.faro.utils.tex_table import calculateTEx
 
 import astropy.units as u
 import numpy as np
 
-__all__ = ("TExTableConfig", "TExTableTask")
+__all__ = ("TExTableConfig", "TExTableTask", "NumSourcesConfig", "NumSourcesTask")
+
+
+class NumSourcesConfig(Config):
+    doPrimary = Field(
```

Comment:
This `doPrimary` field is made obsolete by the new selectors approach implemented in this PR. Suggest to reframe this placeholder task (`NumSourcesTask`) and config (`NumSourcesConfig`) in the new approach.

Reply:
This only works if we use `NumSourcesTask` imported from lsst/faro/measurement/TractTableMeasurementTasks (see above comment)

Reply:
Decided to keep NumSources as is for this ticket.